pyproject.toml

```
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "submit_cgap"
-version = "3.1.0.1b1"  # to become 4.0.0
+version = "4.1.0"
```

Why 4.1.0? We have not yet released a 4.0.0.
```suggestion
version = "4.0.0"
```